cmd/gpu-kubelet-plugin/main.go

```diff
@@ -117,6 +124,27 @@ func newApp() *cli.App {
 			Destination: &flags.imageName,
 			EnvVars:     []string{"IMAGE_NAME"},
 		},
+		&cli.IntFlag{
+			Name:        "healthcheck-port",
+			Usage:       "Port to start a gRPC healthcheck service. When positive, a literal port number. When zero, a random port is allocated. When negative, the healthcheck service is disabled.",
+			Value:       -1,
+			Destination: &flags.healthcheckPort,
+			EnvVars:     []string{"HEALTHCHECK_PORT"},
+		},
+		&cli.StringFlag{
+			Name:        "kubelet-registrar-directory-path",
+			Usage:       "Absolute path to the directory where kubelet stores plugin registrations.",
```

Comment:
IIUC, this is the directory that the kubelet _monitors_ for unix domain sockets (files) placed by plugins. 

So, we should change the text to something like

"Absolute path to the directory that the kubelet monitors for unix domain sockets for plugin registration".

Reply:
(the kubelet does not store plugin registrations there, that I believe is a wrong concept)